package.json

```diff
@@ -24,13 +24,13 @@
     "semantic-release": "semantic-release"
   },
   "dependencies": {
-    "@typescript-eslint/utils": "^5.57.0",
+    "@typescript-eslint/utils": "^6.0.0",
     "tslib": "^2.3.1",
     "tsutils": "^3.21.0"
   },
   "peerDependencies": {
-    "eslint": "^6.0.0 || ^7.0.0 || ^8.0.0",
-    "typescript": "^3.7.5 || ^4.0.0 || ^5.0.0"
+    "eslint": "^7.0.0 || ^8.0.0",
+    "typescript": "^4.2.4 || ^5.0.0"
```

Comment:
Since we are dropping TS v3 and ESLint v6 support we should also remove a BC tests for these versions in the [Github Workflows](https://github.com/gund/eslint-plugin-deprecation/blob/df4dc2319d058ac0b367d71abe7585624b9510ce/.github/workflows/test.yml#L41).
Can you please update those?

Reply:
Yeah, of course! Should I also add eslint 8 and typescript 5 to testing?

Reply:
It's already tested by default so no need